tools/cldr-apps/src/main/webapp/surveytool.css

```diff
@@ -1689,6 +1689,7 @@ table#voteinfo thead th:first-child {
 /* TODO: should load this locally via webpack, see https://unicode-org.atlassian.net/browse/CLDR-9195 */
 @import url("//fonts.googleapis.com/css?family=Noto+Sans:400italic,700italic,400,700");
 @import url("//fonts.googleapis.com/css?family=Noto+Sans+Symbols:400italic,700italic,400,700");
+@import url("//fonts.googleapis.com/css?family=Noto+Nastaliq+Urdu");
```

Comment:
There's a merge conflict here, since in the main branch the @import lines have been moved to the top of the file, as required by modern browsers. The solution is trivial.

Reply:
@btangmu the ticket talks about a wider issue that I'm working on addressing from the UTC side. Current report in https://www.unicode.org/L2/L2023/23273r-kashmiri-yeh.pdf

for CLDR should I file a separate issue?

Reply:
Yes, that would be good. Note that if the deunification option is chose, we would not take data with the new character for some time, waiting for fonts be supported on platforms.

Reply:
It's unlikely that the deunification option will be chosen at this point: this means that there is no further action from CLDR on Kashmiri besides switching the font.